### ISSUE TITLE
Cannot reproduce the results of the paper using PyTorch

### ISSUE THREAD

**Issue by PanosAntoniadis** [behzadhsni/BReG-NeXt#9]
Hello! Awesome work!

I am trying to train the model using the provided code in PyTorch. But I cannot achieve the accuracy mentioned in the paper. Is there something missing? Are the training data the ones that are provided [here](http://mohammadmahoor.com/affectnet/) or have you done further pre-processing on the input images? Also, in which epoch does the model achieve 68.5% accuracy on the categorical model?

Thank you in advance.
